docs/en/setup/README.md

```diff
@@ -25,6 +25,12 @@ You can go to their project repositories for additional info about guides and re
 
 - [SkyAPM GO2Sky](https://github.com/SkyAPM/go2sky). See GO2Sky project document for more details.
 
+## Browser agent
+
+- Activate the [browser-receiver](backend/backend-receivers.md#browser-receiver) following the document.
```

Comment:
I think `receiver-browser` should be activated by default, right? Even the UI is not ready, but it should be there because this will be a part of UI when it is ready.

Change this to `Find browser-receiver setup in the backend receiver doc.` as the 2nd point.

Reply:
done